src/LogExpert/Controls/LogTabControl.cs

```diff
@@ -19,6 +19,12 @@ public LogTabControl()
             //SetStyle(ControlStyles.AllPaintingInWmPaint, true);
             //SetStyle(ControlStyles.UserPaint, true);
             //SetStyle(ControlStyles.DoubleBuffer, true);
+
+            //foreach(TabPage tabPage in base.TabPages)

```

Comment:
are the for testing purposes commented, or any other reason?
if the need to be there, please write a short comment why its here

Reply:
Removed